src/bin/nitrogen.rs

```diff
@@ -82,13 +82,12 @@ enum Commands {
     },
 
     Start {
+        /// Name of the CloudFormation stack/provisioned EC2 instance
         name: String,
-
-        /// EC2 key-pair to use for the provisioned instance
-        key_name: String,
-
-        /// EC2 key-pair to use for the provisioned instance
-        public_key: String,
+        /// File of public key to be used for ssh with the provisioned instance
+        public_key_file: String,
+        /// File of private key to be used for ssh
+        private_key: String,
```

Comment:
Should we make these names similar? maybe 'public_key' and 'private_key'?

Reply:
Makes sense. We need to do a full clean up of var names. I'm fine to merge this as is and clean them up separately